app/views/projects/show.html.erb

```diff
@@ -89,15 +93,27 @@
   </table>
   <%= render partial: "projects/import_export" %>
 
-  <div class="btn-group">
-    <%= link_to "Edit or Delete Project", edit_project_path(@project.id), class: "button magenta"  %>
-
-    <% unless @project.parent_id %>
-      <%= link_to 'Add Sub-Project', project_new_sub_project_path(@project.id), id:"subproject", class: "button magenta" %>
+  <div class="btn-group actions">
+    <% if @project.archived? %>
+      <button class="button magenta disabled" aria-disabled="true" disabled>Edit or Delete Project</button>
+      <button class="button magenta disabled" aria-disabled="true" disabled>Add Sub-Project</button>
```

Comment:
this button is rendered conditionlly if not archived, it should be rendered conditionally if archived too